### PR TITLE
Added documentation for manually creating column via migrations.RunSQL

### PR DESCRIPTION
**Pull request by gen1us2k**

### CODE REVIEW

**Comment by gen1us2k** [comment]
Any news? Comments?

File: docs/ref/migration-operations.txt [hunk 1]
```diff
@@ -232,6 +232,25 @@ manually creating a column, you should pass in a list containing an ``AddField``
 operation here so that the autodetector still has an up-to-date state of the
 model (otherwise, when you next run ``makemigrations``, it won't see any
 operation that adds that field and so will try to run it again).
+For example::
```
Comment (timgraham) [inline docs/ref/migration-operations.txt]: please flow this on the previous line

File: docs/ref/migration-operations.txt [hunk 2]
```diff
@@ -232,6 +232,25 @@ manually creating a column, you should pass in a list containing an ``AddField``
...
+
+    migrations.RunSQL(
+        "ALTER TABLE musician "
```
Comment (timgraham) [inline docs/ref/migration-operations.txt]: could add an `app_label` to the table name.
could we simplify the example by adding a single field?

File: docs/ref/migration-operations.txt [hunk 3]
```diff
@@ -232,6 +232,25 @@ manually creating a column, you should pass in a list containing an ``AddField``
...
+            migrations.AddField(
+                'musician',
+                'name',
+                models.CharField(max_length=255),
+            ),
+            migrations.AddField(
+                'musician',
+                'genre',
+                models.CharField(max_length=128),
+            )
```
Comment (timgraham) [inline docs/ref/migration-operations.txt]: include a trailing comma

**Comment by gen1us2k** [comment]
Updated diff

**Comment by timgraham** [comment]
merged in a1d0c60fa05cbad2e5a25ec37e0afaf1b84c9302, thanks!

**Pull request closed without merging**